src/WebJobs.Extensions.DurableTask/Listener/TaskOrchestrationShim.cs

```diff
@@ -213,6 +213,10 @@ private async Task HandleOutOfProcExecution(JObject executionResult)
             {
                 this.Context.SetOutput(execution.Output);
             }
+            else
+            {
+                await Task.Delay(-1);

```

Comment:
Let's add a comment here explaining why we're doing this, lest we forget in the future.

Reply:
Done.